### PR TITLE
CI: unify CPU tests

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

Merge together a package and full testing...
Consider if we pull cache before installing itself won't leak some dependencies from devel.txt

## Before submitting

- [ ] Was this **discussed/approved** via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/lightning-bolts/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does **only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes?
- [ ] Did you write any **new necessary tests**? \[not needed for typos/docs\]
- [ ] Did you verify new and **existing tests** pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/lightning-bolts/blob/master/CHANGELOG.md)?

## PR review

- [x] Is this pull request **ready for review**? (if not, please submit in draft mode)

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def _prepare_extras():
     include_package_data=True,
     zip_safe=False,
     keywords=["deep learning", "pytorch", "AI"],
-    python_requires=">=3.7",
+    python_requires=">=3.8",
```
Comment (otaj) [inline setup.py:59]: Why was this needed? 3.7 still has eight months of life left

Comment (Borda) [reply setup.py:59]: ok :)

Comment (Borda) [reply setup.py]: on the other hand py3.10 does not work :D

File: .github/workflows/ci-tests.yml [hunk 2]
```diff
@@ -43,10 +47,27 @@ jobs:
...
+
+    - name: Install package
+      run: |
+        pip install -e . --upgrade --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
+        pip list
+
+    - name: Test Package [only]
+      run: |
+        pip install pytest -q
+        # TODO: package shall be fine to run full without anhy ignores
```
Comment (akihironitta) [inline .github/workflows/ci-tests.yml:65]:
```suggestion
        # TODO: package shall be fine to run full without any ignores
```

**Review by akihironitta — APPROVED** [review]
